tests/integration/suite/test_catalog.py

```diff
@@ -7,6 +7,20 @@
 from .conftest import set_server_version
 
 
+def wait_for_atleast_workload(pclient, nsid, timeout=60, count=0):
+    start = time.time()
+    interval = 0.5
+    workloads = pclient.list_workload(namespaceId=nsid)
+    while len(workloads.data) < count:
+        if time.time() - start > timeout:
+            print(workloads)
```

Comment:
print ?

Reply:
This can help with debugging. Only throws logs on an error